Shared/ViewModels/ItemViewModel/ItemViewModel.swift

```diff
@@ -90,22 +90,44 @@ class ItemViewModel: ViewModel, Stateful {
         super.init()
 
         // TODO: should replace with a more robust "PlaybackManager"
-        Notifications[.itemMetadataDidChange].publisher
-            .sink { [weak self] notification in
-                if let userInfo = notification.object as? [String: String] {
-                    if let itemID = userInfo["itemID"], itemID == item.id {
-                        Task { [weak self] in
-                            await self?.send(.backgroundRefresh)
-                        }
-                    } else if let seriesID = userInfo["seriesID"], seriesID == item.id {
-                        Task { [weak self] in
-                            await self?.send(.backgroundRefresh)
-                        }
-                    }
-                } else if let newItem = notification.object as? BaseItemDto, newItem.id == self?.item.id {
-                    Task { [weak self] in
-                        await self?.send(.replace(newItem))
-                    }
+//        Notifications[.itemMetadataDidChange].publisher
```

Comment:
Do we need this commented code or can we delete this?

Reply:
Good catch on this one, forgot to clean up.